src/badge.jsx

```diff
@@ -41,80 +93,40 @@ const Badge = React.createClass({
      */
     style: React.PropTypes.object,
   },
+
```

Comment:
Good point! Thanks for reminding me. I'd say 80+% of the components I've switched so far are candidates/ready to be switched to classes using the codemod :+1: